js/components/Editor.css

```diff
@@ -5,4 +5,346 @@
 .editor.breakpoint {
   fill: var(--theme-selection-background);
   width: 40px;
+}
+
+/* vim:set ts=2 sw=2 sts=2 et: */
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/* todo: maybe add later? */
+/*@import url(variables.css);*/
+/*@import url(common.css);*/
+/*@import url(toolbars.css);*/
```

Comment:
@matt-oconnell did you delete the other import/url references in the file? If so, could you comment them out instead so it's clear what we need to move over to have the full functionality